client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx

```diff
@@ -156,6 +160,28 @@ const InvestigationTable: React.FC = (): JSX.Element => {
         return (<Popper {...props} style={{ width: 350 }} placement='bottom-start' />)
     }
 
+    const RowTooltip = (props: React.PropsWithChildren<{creationDate: Date; startTime: Date;}>) => {
+        const {creationDate, startTime} = props;
+        const noDataMessage = 'אין מידע אודות תאריכים לחקירה זו';
+        const formatDate = (date: Date) => format(new Date(date), 'dd/MM/yyyy');
+
+        const title = <>
```

Comment:
I think using `useMemo` here with creationDate and startTime is pretty good.

Reply:
sure